### DISCUSSION TITLE
Historical Q: When were INPFC areas first used to design the GOA survey?

### DISCUSSION THREAD

**Discussion by MargaretSiple-NOAA** [MargaretSiple-NOAA/goa-ai-data-reports#74] [General]
Text of GOA methods:

> The survey assessed only that portion of the slope between 200 and 700 m, which represented 18% of the total survey area. The survey was originally stratified by statistical areas erected by the former International North Pacific Fisheries Commission (INPFC).

From Ned:
"Would be good for me to figure out when "initially" was. help to understand whether we retroactively applied these areas to previous surveys or of the INPFC history started with the GOA history."

Tagging @zoyafuso-NOAA because he might know.

**Comment by zoyafuso-NOAA** [comment]
Hey Megsie and Ned,
I don’t know this answer. The best that I could do was to search through past Tech Memos about our early surveys and here’s what I found. First, the INPFC was established in the early 1950s so I can envision the cooperative surveys in those GOA early years being linked to the INPFC areas from the get-go. Munro and Hoff (1995) details the 1984 and 1987 GOA BTS and is the first year where the current stratification as we know it was described, which consists of the combination of INPFC area ,depth bin, and shelf/gully/slope. Prior to the 1984 survey is a 1978 survey where INPFC areas were used, but within each INPFC area, only 5 depth strata were defined between 0-500 m, 100-m intervals. And so, presumably there were 25 total strata in that survey. The exploratory surveys between 1948-1965 reference the INPFC areas when summarizing catch statistics but the exploratory surveys done in this time period did not follow a sampling design and used smaller, pre-defined West-East regions that we don’t use anymore (see map and table below).


 
 
![image](https://github.com/MargaretSiple-NOAA/goa-ai-data-reports/assets/60302777/f51573e1-750f-47f9-8b8f-6e562c651b40)

![image](https://github.com/MargaretSiple-NOAA/goa-ai-data-reports/assets/60302777/f6288fc1-f25b-43b2-af1b-63ec2a9e03c8)

[GOA_Exploratory.pdf](https://github.com/MargaretSiple-NOAA/goa-ai-data-reports/files/13446280/GOA_Exploratory.pdf)

[GOA_1978.pdf](https://github.com/MargaretSiple-NOAA/goa-ai-data-reports/files/13446246/GOA_1978.pdf)

[GOA_1980s.pdf](https://github.com/MargaretSiple-NOAA/goa-ai-data-reports/files/13446248/GOA_1980s.pdf)

**Comment by MargaretSiple-NOAA** [comment]
Thank you Zack! And sorry it took so long for me to respond to this. I appreciate the deep dive. @Ned-Laman-NOAA take a look at the above and see if this answers your question. Also I don't ever know where to put this "history of the survey" material... currently I have a folder of scraps on my laptop with notes of what people have told me.

**Comment by Ned-Laman-NOAA** [comment]
Thanks to you both. This does answer my question. It seems we've been using or taking INPFC areas into account in the Gulf survey since it's inception in 1953.
Emily has a history document [here](https://docs.google.com/document/d/1brkJ1qgmts2nJ-p7ugsqfMneDeuXiwbhyzmvd3HT3MA/edit#heading=h.w666wh7e438k) that tracks GAP Survey histories and gets served up in our web pages. I put a bare bones bit of information in there for our two surveys, but would welcome your contribution of any additional discoveries or useful historic information. This seems a good place for the bits of history you've been collecting.

**Comment by MargaretSiple-NOAA** [comment]
I'll add the notes I have from my own google doc, which I just re-remembered 5 mins ago. Thanks Ned

**Comment by Ned-Laman-NOAA** [comment]
Here's another tidbit I ran across:

The International North Pacific Fisheries Commission (INPFC) existed from 1952 to 1993 to facilitate the coordination, cooperation, and exchange of scientific information for marine species. It was replaced by the [North Pacific Anadromous Fish Commission (NPAFC)](https://uia.org/s/or/en/1100003900).  The INPFC established reporting regions that were used to stratify the Aleutian Islands and Gulf of Alaska bottom trawl surveys and have been kept for consistency to the present (2024). 

I believe that the West Coast NMFS Groundfish survey also uses historic INPFC districts for stratifying those surveys.
